### ISSUE TITLE
pre Loading screen crash

### ISSUE THREAD

**Issue by cixuthewanderer** [CritLoren/Yggdrasil-VR#66]
**Describe the bug**
a fair amount of the time I crash after entering an area that needs a loading screen

**To Reproduce**
1. I enter an area or save
2. The game crashes

**Expected behavior**
the game crashes upon entering any new areas, the save crash is less often

**Crash log**
If applicable, add a save to reproduce your problem. You can find them at `My Documents/My Games/Skyrim VR/SKSE/` where they'll be named something like `crash-2021-12-20-23-21-37.log`.

**Desktop (please complete the following information):**
 - Yggdrasil VR version: 2.0-5
 - Storage type Yggdrasil VR is installed on:  SSD
 - OS version: Windows 11 
 - RAM: 32gb
 - CPU: AMD Ryzen 5900x 12-core processor
 - GPU: Nvidia RTX 3080
 - Antivirus Windows Defender, Bitdefender

[crash-2022-09-19-21-42-55.log](https://github.com/CritLoren/Yggdrasil-VR/files/9603272/crash-2022-09-19-21-42-55.log)
[crash-2022-09-19-22-24-14.log](https://github.com/CritLoren/Yggdrasil-VR/files/9603273/crash-2022-09-19-22-24-14.log)
[crash-2022-09-19-20-38-54.log](https://github.com/CritLoren/Yggdrasil-VR/files/9603274/crash-2022-09-19-20-38-54.log)
[crash-2022-09-19-21-11-21.log](https://github.com/CritLoren/Yggdrasil-VR/files/9603275/crash-2022-09-19-21-11-21.log)
[crash-2022-09-19-21-22-46.log](https://github.com/CritLoren/Yggdrasil-VR/files/9603276/crash-2022-09-19-21-22-46.log)

**Comment by CritLoren** [comment]
Looks like the VFS keeps failing, did you make sure to add the Yggdrasil folder to exclusions in your antivirus?

**Comment by cixuthewanderer** [comment]
no, added it now. I'll test it tomorrow.

**Comment by cixuthewanderer** [comment]
[crash-2022-09-21-21-58-17.log](https://github.com/CritLoren/Yggdrasil-VR/files/9637627/crash-2022-09-21-21-58-17.log)


still ran into a crash walking through a door, seeing if i added the right exceptions and retrying

**Comment by cixuthewanderer** [comment]
crashed again with no log at \Documents\my games\Skyrim VR\SKSE

am I going to the wrong log location?

**Comment by CritLoren** [comment]
can you try a reinstall? that last log you posted refers to a mesh, and I don't see a reason as to why it'd be causing crashes.

**Comment by CritLoren** [comment]
No, that's where it should be, if logs aren't generated something isn't going well, did you try reinstalling?

**Comment by cixuthewanderer** [comment]
Yeah I'm going to launch it here soon, was having trouble with the install but you helped me earlier in the discord.

**Comment by cixuthewanderer** [comment]
[crash-2022-09-24-23-37-32.log](https://github.com/CritLoren/Yggdrasil-VR/files/9639752/crash-2022-09-24-23-37-32.log)

**Comment by cixuthewanderer** [comment]
door issue seems resolved but crash on save is still happening

**Comment by cixuthewanderer** [comment]
door one was by far worse so this is definitely better

**Comment by CritLoren** [comment]
Still very much looks like a VFS issue, can you temporarily disable real time scanning in Bitdefender and see if that helps?
